### PR TITLE
fix: vertical alignment in cards

### PR DESCRIPTION
**Pull request by TopETH**
## Summary by CodeRabbit

- **Style**
	- Updated styling on the `AuctionCard` component to enhance layout and responsiveness.
- **Refactor**
	- Added an export statement for a new module in the `smartsale-db` package.

### CODE REVIEW

**Review by AndlerRL — APPROVED** [review]
LGTM!

File: apps/webapp/components/routes/home/auction-card.tsx
Comment (AndlerRL) [inline apps/webapp/components/routes/home/auction-card.tsx:1]: I recommend to push the action button wrapper to the bottom by giving the `mt-auto`. On smaller screens we might get the same issue. 

Also continue adding a number to this file to make the build happen after merge -> https://github.com/bitcashorg/smartsale/blob/develop/packages/smartsale-db/src/client.ts#L5

Comment (TopETH) [reply apps/webapp/components/routes/home/auction-card.tsx:1]: image
I think we already have it there. @AndlerRL Could you please check it out again?

**Review by gaboesquivel — APPROVED** [review]

**Comment by AndlerRL** [comment]
[Development Support, Nathan - April 2024, 1 of 2](https://trello.com/c/p7qqDJXX/26-development-support-nathan-april-2024-1-of-2)

**Pull request merged**